Cargo.toml

```diff
@@ -5,7 +5,7 @@ version = "0.19.0"
 keywords = ["incremental", "parsing", "julia"]
 categories = ["parsing", "text-editors"]
 repository = "https://github.com/tree-sitter/tree-sitter-javascript"
```

Comment:
This URL is wrong 😬 . Could you write the correct URL in this same PR?

Reply:
Does the latest commit fix the issue?